checks/default.nix

```diff
@@ -3,185 +3,217 @@
-      } ];
-    } ]
+      }];
+    }
+      {
+        id = "qemu-tcg";
+        modules =
+          let
+            # Emulate a different guest system than the host one
+            guestSystem =
+              if "${system}" == "x86_64-linux" then "aarch64-unknown-linux-gnu"
+              else "x86_64-linux";
+          in
+          [
+            {
+              microvm = {
+                hypervisor = "qemu";
+                # Force the CPU to be something else than the current
+                # system, and thus, emulated with qemu's Tiny Code Generator
+                # (TCG)
+                cpu =
+                  if "${system}" == "x86_64-linux" then "cortex-a53"
+                  else "Westmere";
+              };
+              nixpkgs.crossSystem.config = guestSystem;
+            }
+          ];
+      }
+      {
+        id = "cloud-hypervisor";
+        modules = [{
+          microvm.hypervisor = "cloud-hypervisor";
+        }];
+      }
+      {
+        id = "crosvm";
+        modules = [{
+          microvm.hypervisor = "crosvm";
+        }];
+      }
+      {
+        id = "firecracker";
+        modules = [{
+          microvm.hypervisor = "firecracker";
+        }];
+      }
+      {
+        id = "kvmtool";
+        modules = [{
+          microvm.hypervisor = "kvmtool";
+        }];
+      }]
     # ro-store
-    [ {
+    [{
       # squashfs/erofs
       id = null;
-    } {
-      # 9pfs
-      id = "9pstore";
-      modules = [ ({ config, ... }: {
-        microvm = {
-          shares = [ {
-            proto = "9p";
-            tag = "ro-store";
-            source = "/nix/store";
-            mountPoint = "/nix/.ro-store";
-          } ];
-          testing.enableTest = builtins.elem config.microvm.hypervisor [
-            # Hypervisors that support 9p
-            "qemu" "crosvm" "kvmtool"
-          ];
-        };
-      }) ];
-    } ]
+    }
+      {
+        # 9pfs
+        id = "9pstore";
+        modules = [
+          ({ config, ... }: {
+            microvm = {
+              shares = [{
+                proto = "9p";
+                tag = "ro-store";
+                source = "/nix/store";
+                mountPoint = "/nix/.ro-store";
+              }];
+              testing.enableTest = builtins.elem config.microvm.hypervisor [
+                # Hypervisors that support 9p
+                "qemu"
+                "crosvm"
+                "kvmtool"
+              ];
+            };
+          })
+        ];
+      }]
     # rw-store
-    [ {
+    [{
       # none
       id = null;
-    } {
-      # overlay volume
-      id = "overlay";
-      modules = [ ({ config, ... }: {
-        microvm.writableStoreOverlay = "/nix/.rw-store";
-        microvm.volumes = [ {
-          image = "nix-store-overlay.img";
-          label = "nix-store";
-          mountPoint = config.microvm.writableStoreOverlay;
-          size = 128;
-        } ];
-      }) ];
-    } ]
+    }
+      {
```

Comment:
These curly braces being on different indentation levels illustrates well why I don't like any of the current formatters.